wemake_python_styleguide/violations/consistency.py

```diff
@@ -2102,3 +2104,33 @@ class IterableUnpackingViolation(ASTViolation):
 
     error_template = 'Found an unnecessary iterable unpacking'
     code = 356
+
+
+@final
+class LineCompriseCarriageReturnViolation(TokenizeViolation):
+    """
+    Forbids to use \r (carriage return) in line breaks.
+
+    Reasoning:
+        We enforce strict consitency rules about how to break lines in 'str' type.
+        In this case, we only use newlines (\n), not carriage returns.
+        PEP8 doesn't yet replace \r in strings, however obsolete
+        'Windows'-style line breaks not allowed in code.
+
+    Solution:
+        Use only ``\n`` (not ``\r\n`` or ``\r``) to break lines in strings.
+
+    Example:
```

Comment:
Please, remove example. It does not make sense here.

Reply:
Ok, 'll fix it soon

Reply:
fixed